### PR TITLE
Update stack.json

### PR DESCRIPTION
**Pull request by alem355**

### CODE REVIEW

File: static/stack.json [hunk 1]
```diff
@@ -129,161 +129,161 @@
...
       "Hold": "<code class='lang'>Selenuim</code> <code class='lang'>Rest-Assured</code> <code class='lang'>Gemini</code> <code class='lang'>Nightwatch</code>"
     },
     {
       "Area": "QA",
       "Cases": "Performance",
       "Research": "",
-      "Trial": "<code class='lang'>Yandex.Tank</code>",
-      "Best choice": "<code class='lang'>Scala</code> <code class='lang'>Gatling</code> <code class='lang'>Lighthouse</code>",
-      "Hold": "<code class='lang'>Artillery</code> <code class='lang'>Jmeter</code>"
+      "Trial": "<code class='tool'>Yandex.Tank</code>","<code class='tool'>HP Performance centre</code>",
```
Comment (dsugrobov) [inline static/stack.json:182]: bug here

File: static/stack.json [hunk 2]
```diff
@@ -129,161 +129,161 @@
...
+      "Best choice": "<code class='lang'>Scala</code> <code class='tool'>Gatling</code> <code class='tool'>Lighthouse</code>",
+      "Hold": "<code class='tool'>Artillery</code> <code class='tool'>Jmeter</code>"
     },
     {
       "Area": "QA",
       "Cases": "Infrastructure & Documentation",
-      "Research": "<code class='lang'>Moon</code>",
+      "Research": "<code class='tool'>Moon</code>",
```
Comment (dsugrobov) [inline static/stack.json:189]: `moon` twice here

File: static/stack.json [hunk 3]
```diff
@@ -292,87 +292,87 @@
...
       "Hold": ""
     },
     {
       "Area": "Infrastructure",
       "Cases": "CDN",
       "Research": "",
       "Trial": "",
-      "Best choice": "<code class='lang'>Ngnix</code>",
-      "Hold": "<code class='lang'>Cloudinary</code>"
+      "Best choice": "<code class='platform'>Ngnix</code>",
```
Comment (dsugrobov) [inline static/stack.json:367]: Ngenix here :(

Comment (alem355) [reply static/stack.json:367]: What do you mean?

Comment (alem355) [reply static/stack.json:367]: got it

**Pull request merged**